src/libraries/System.Console/src/System/ConsolePal.WebAssembly.cs

```diff
@@ -74,6 +76,9 @@ public override void Flush()
 
     internal static class ConsolePal
     {
+        private static bool s_consoleInitialized;
```

Comment:
If we expect it may ever be possible for this code to be accessed concurrently from multiple threads, s_consoleInitialized should be marked volatile.